### DISCUSSION TITLE
Nowy feature branch

### DISCUSSION THREAD

**Discussion by emkarcinos** [emkarcinos/WMIAdventure#422] [Organisation of work]
Zrobiłem nowego brancha `CardEditor_Description` żebyśmy mogli na niego mergować wszystkie taski które robimy w ramach aktualizacji karty o ten automatycznie generowany opis. Na maina się zmerguje jak wszystko będzie gotowe.

Starajcie się na przyszłość też robić takie gałęzie, które będą zbiorcze dla wszystkich tasków dotyczących jednego featura. Za mało dbamy o naszego maina i wszystko tam mergujemy, a powinnismy właściwie mergować tam kompletne ukończone funkcje.

**Comment by itchystone** [comment]
Racja, to w sumie jak było wiele drobnych tasków dotyczących poprawy wyglądu i animacji edytora kart to powinniśmy pewnie stworzyć feature branch `CardEditor_better_looks`, czy coś w tym stylu

**Reply by emkarcinos** [reply]
Tak, o takie coś mi chodzi.
Widziałem, że w większości repozytoriów tak robią, a na mainie jest już naprawdę w pełni działający kod stanowiący jakąś wersję.
